app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_count(post)
+    count = post.likes.count
+    post.update(likes_counter: count)
+  end
```

Comment:
- [ ] Optional suggestion

I would recommend doing something like this to count the posts.

```
def  self.update_count
  post.increment!(:likes_counter)
end
```